contracts/Pool.sol

```diff
@@ -36,6 +36,8 @@ contract Pool is PoolFDT {
     address public immutable stakeLocker;      // Address of the StakeLocker, escrowing stakeAsset
     address public immutable superFactory;     // The factory that deployed this Loan
 
+    IMapleGlobals private immutable globals;
```

Comment:
Keep `globals` in state to reduce cost of fetching it.

Reply:
We make it fetchable on purpose so that we can setGlobals in the PoolFactory and it will affect the change across all pools

Reply:
Ah, in which case I will undo that then. 700 gas is back on the menu...